external/mmdetection/configs/custom-object-detection/gen3_mobilenetV2_SSD/hpo_config.yaml

```diff
@@ -1,9 +1,9 @@
 metric: mAP
-search_algorithm: smbo
+search_algorithm: asha
 early_stop: None
 hp_space:
   learning_parameters.learning_rate:
-    param_type: quniform
+    param_type: qloguniform
     range:
       - 0.001
       - 0.1
```

Comment:
if you choose base as hp search space, modifying batch size to base is also needed.